crates/ruff_linter/src/rules/ruff/rules/missing_fstring_syntax.rs

```diff
@@ -203,7 +217,8 @@ fn should_be_fstring(
 fn has_brackets(possible_fstring: &str) -> bool {
     // this qualifies rare false positives like "{ unclosed bracket"
     // but it's faster in the general case
-    memchr2_iter(b'{', b'}', possible_fstring.as_bytes()).count() > 1
+    let mut searcher = memchr2_iter(b'{', b'}', possible_fstring.as_bytes());
+    searcher.next().and_then(|_| searcher.next()).is_some()
```

Comment:
This is a micro-optimisation opportunity I spotted. I believe `count()` will consume the whole iterator, which shouldn't be necessary for our purposes: we can short-circuit as soon as we've found 2

Reply:
Nit: You could probably do `searcher.nth(1).is_some()`